packages/firestore/src/local/shared_client_state.ts

```diff
@@ -830,6 +852,13 @@ export class WebStorageSharedClientState implements SharedClientState {
               return this.handleOnlineStateEvent(onlineState);
             }
           }
+        } else if (event.key === SEQUENCE_NUMBER_KEY) {
+          if (this.sequenceNumberListener) {
+            const sequenceNumber = parseSequenceNumber(event.newValue);
```

Comment:
Nit: Can you follow the existing naming conventions in this file (`this. fromLocalStorageOnlineState` for example).

Reply:
Sure, but there doesn't seem to be a good reason to make this a method rather than just a function. It doesn't rely on any state from the instance.

Reply:
Renamed to `fromLocalStorageSequenceNumber`

Reply:
Two reasons to make this a method:

- It namespaces the method to the class that deals with Local Storage. You can then drop `LocalStorage` from the name, and it is immediately obvious that this method is only useful to one of the two classes in this file.
- It's consistent with the rest of the file. The new name is already a big step in that direction (thanks!)